### ISSUE TITLE
mixin chain functions not work

### ISSUE THREAD

**Issue by iml885203** [sweetalert2/sweetalert2#2174]
mixin chain function is very useful when multiple extend.
mixin chain function is work on v9.10.11.
but it not work when I upgrade to v10.15.4.

example: 
```javascript
let myMixin = Swal.mixin({title: 'title1'});
myMixin.fire(); // show title1
let myMixin2 = Swal.mixin({title: 'title2'});
myMixin2.fire(); // show title1, can not change to title2
```

another simple example:
```javascript
Swal.mixin({title: 'title1'}).mixin({title: 'title2'}).fire(); // show title2 on v9.10.11
Swal.mixin({title: 'title1'}).mixin({title: 'title2'}).fire(); // show title1 on v10.15.4
```

**Comment by zenflow** [comment]
I bisected the repo to find the regression was introduced in v10.12.4 with PR #2133
I have already put in a PR for a quick fix #2175

**Comment by limonte** [comment]
Thanks for reporting @iml885203 and kudos to @zenflow for fixing the issue 🍻
